apps/theme/app/themebuilder/page.tsx

```diff
@@ -1,88 +1,65 @@
 'use client';
 
 import type { ColorMode } from '@digdir/designsystemet';
-import { Heading, Link, Tabs } from '@digdir/designsystemet-react';
-import { ChevronLeftIcon } from '@navikt/aksel-icons';
-import NextLink from 'next/link';
+import { Heading } from '@digdir/designsystemet-react';
+
 import { useRouter } from 'next/navigation';
 import {
   BorderRadius,
   ColorContrasts,
   ColorPreview,
   ColorTokens,
   Colors,
+  Header,
   Sidebar,
+  ToggleChip,
 } from '../../components';
 import { Toggle } from '../../components/Toggle/Toggle';
 import { useThemeStore } from '../../store';
 import { useThemeParams } from './_utils/useThemeParams';
 import classes from './page.module.css';
 
 export default function Home() {
-  const colors = useThemeStore((state) => state.colors);
-  const appearance = useThemeStore((state) => state.appearance);
   const setActivePage = useThemeStore((state) => state.setActivePage);
   const router = useRouter();
   const activePage = useThemeStore((state) => state.activePage);
   const setAppearance = useThemeStore((state) => state.setAppearance);
+  const items = [
+    {
+      name: 'Navn på tema',
+      value: 'intro',
+    },
+    {
+      name: 'Farger',
+      value: 'colors',
+    },
+    {
+      name: 'Border radius',
+      value: 'radius',
+    },
+  ];
```

Comment:
This can be moved outside of the component. Maybe we can share this between the different pages as well?